apps/web/src/components/mapping/google-maps/map-view.tsx

```
@@ -153,88 +166,80 @@ const Mapview = () => {
+            } else {
+              resolve(null);
             }
-          );
+          });
+        });
+        
+        const bestMatch = response?.find(result => {
+          return result.types.includes("locality");
         });
+
+        if (bestMatch && bestMatch.place_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)